OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -353,6 +353,7 @@ public interface Requires<T> where T : class, ITraitInfoInterface { }
 	public interface UsesInit<T> : ITraitInfo where T : IActorInit { }
 
 	public interface INotifySelected { void Selected(Actor self); }
+	public interface INotifyDeselected { void Deselected(Actor self); }
```

Comment:
This could live as a method inside `INotifySelected`.  It doesn't make much sense for something to implement one but not the other.